extensions/TheShovel/ShovelUtilsTW.js

```diff
@@ -0,0 +1,257 @@
+console.log("ShovelUtils v1.1")
+tempImageLoad = null
+string = null;
+temp = null;
+tempVAR = null;
+
+
+
+(function(Scratch) {
```

Comment:
```suggestion
(function(Scratch) {
  'use strict';
```
fixes use the function form of use strict